src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,103 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  max-width: getContainerWidth($card-width, $card-gap, 40);
+  padding: 50px 40px;
+
+  &--centered {
+    margin: 0 auto;
+  }
+
+  &__card {
+    margin-right: $card-gap;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
```

Comment:
I suggest to add a variable for this color. Looks like you use it only in once place, but still if you'll need to change it you will have to look for the place where it is used.

Reply:
Thank you for the review. I've amended my code.